.github/hooks/post-commit-unix.sh

```
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+# Omega_KG Universal Post-Commit Hook
+# Auto-detects environment (Git Bash/WSL2/Linux) and logs to Obsidian
+
+# Omega_KG Hook Version
+HOOK_VERSION="1.0.0"
```

Shellcheck (0.11.0)

[warning] HOOK_VERSION appears unused. Verify use (or export if used externally).

(SC2034)